### DISCUSSION TITLE
Probleme mit Datenpunkt Eigenschaft

### DISCUSSION THREAD

**Discussion by lupolio** [Zefau/ioBroker.jarvis#1748] [Q&A]
Hallo Community,

ich mache meine ersten Versuche mit iobroker / jarvis und komme trotz tagelanger Recherche nicht weiter.
Folgendes Setup:
ZigBee Steckdose <-> zigbee2mqtt <-> mosquitto <-> iobroker

Die Kommunikation funktioniert generell. Ich habe unter Objects einen Eintrag "state", der sich ändert, wenn die Steckdose geschaltet wird. Der "state" ist ein String und kann entweder "ON" oder "OFF" sein. 
![image](https://user-images.githubusercontent.com/42739710/175544263-079988a4-f1db-47ba-8bfb-7c8994912f0a.png)

Wenn ich den Eintrag in iobroker manuell überschreibe, wird ebenfalls die Steckdose geschaltet :)

Mein Problem kommt ins Spiel, jarvis beizubringen den Wert in "state" korrekt zu lesen / schreiben.
So sieht meine Konfiguration aus:
![image](https://user-images.githubusercontent.com/42739710/175544063-262c8491-7d34-4384-baa1-ffbfb1e08a04.png)

Der Switch verhält sich im Widget aber so:
![image](https://user-images.githubusercontent.com/42739710/175544104-c72dad5c-d50b-4ad5-b4aa-ce4c308f2702.png)
![image](https://user-images.githubusercontent.com/42739710/175544153-807b3759-8412-4136-aba4-46ac66447ed0.png)

"OFF" wird korrekt in den "state" zurückgeschrieben, aber statt "ON" kommt nur ein "true" im state an.

Hat jemand zufällig eine Idee, was ich falsch mache? 

Danke !!!

Ach so: Ich nutze Jarvis 3.0.11 und iobroker 5.3.8

**Comment by lupolio** [comment]
Noch ein interessanter Nachtrag: 
wenn ich im iobroker den Wert für "state" manuell auf "ON" setze, dann registriert jarvis das auch - die Anzeige sieht aber auch "komisch" aus:
![image](https://user-images.githubusercontent.com/42739710/175547719-f16e67e9-47a3-4398-bce1-022193f0dc2f.png)

**Comment by Zefau** [comment]
Ist ein Bug: https://github.com/Zefau/ioBroker.jarvis/issues/1626
